HashimX.CoolMath.Library.Tests/CalculatorTests.cs

```diff
@@ -0,0 +1,83 @@
+using HashimX.CoolMath.library;
+
```

Comment:
Well done. great work and all looks great. even it is your first try. Just remove extra line spaces on your next commit.